daemon/src/supervisor.rs

```diff
@@ -13,6 +13,13 @@ pub struct Actor<T, R> {
     ctor: Box<dyn Fn(Address<Self>) -> T + Send + 'static>,
     tasks: Tasks,
     restart_policy: Box<dyn FnMut(R) -> bool + Send + 'static>,
+    metrics: Metrics,
+}
+
+#[derive(Default, Clone, Copy)]
+pub struct Metrics {
+    /// How many times the supervisor spawned an instance of the actor.
```

Comment:
looking at the test it seems like like this is the number of messages the actor has handled?

Reply:
Nope, if you look at the implementation, it is the number of times `spawn_new` has been called! :)